### PR TITLE
Update it-IT.lua

### PR DESCRIPTION
**Pull request by Enryy**
Added DNA, frisk, fingerpint etc...

### CODE REVIEW

File: languages/it-IT.lua [hunk 1]
```diff
@@ -3619,8 +3679,18 @@ OP.Global.Locales.Languages["it-IT"] = {
...
+		slow_motion_disabled = "Disattivata con successo la modalità slow motion.",
+		slow_motion_failed = "Impossibile attivare la modalità slow motion.",
+		slow_motion_missing_perms = "Tentativo di attivare la modalità slow motion senza il permesso appropriato.",
+		slow_motion_enabled_help = "Slow motion mode has been enabled.",
+		slow_motion_disabled_help = "Slow motion mode has been disabled."
+	},
+
 	snow = {
 		hold_to_pick_up_snowballs = "Tieni premuto ~INPUT_CONTEXT~ per prendere una palla di neve."
+
```
Comment (InZidiuZ) [inline languages/it-IT.lua:3693]: Random newline?

File: languages/it-IT.lua [hunk 2]
```diff
@@ -2149,6 +2198,17 @@ OP.Global.Locales.Languages["it-IT"] = {
...
 
+	fingerprint = {
+		taking_fingerprint = "Prendendo impronta digitale",
+		already_fingerprinting = "Stai già prendendo l'impronta digitale di un giocatore.",
+		sample_no_player = "Non ci sono giocatori vicini a cui puoi prendere l'impronta digitale.",
+		sample_no_bags = "Non hai un sacchetto delle prove.",
+		fingerprint_evidence = "Impronta digitale",
+		evidence_failed = "Non sei riuscito a prendere l'impronta digitale del giocatore.",
+		evidence_text = "Tipo di prova: Fingerprint\nFingerprint di ${fullName} #${characterId}\n\nInformazioni aggiuntive:\n • E' stato raccolto alle: ${time}"
+
```
Comment (InZidiuZ) [inline languages/it-IT.lua:2209]: Random newline?

File: languages/it-IT.lua [hunk 3]
```diff
@@ -2131,12 +2167,25 @@ OP.Global.Locales.Languages["it-IT"] = {
...
+	
+	dna_evidence = {
+		taking_sample = "Prelevando campioni di DNA",
+		already_taking_sample = "Hai già preso un campione di DNA di questo giocatore.",
+		sample_no_player = "Non ci sono giocatori vicini per prendere campioni di DNA.",
+		sample_no_bags = "Non hai un sacchetto delle prove.",
+		dna_evidence_bag = "Campione di DNA",
+		evidence_failed = "Non sei riuscito a prendere il campione di DNA.",
+		evidence_text = ": Campione di DNA\nDNA prelevato da ${fullName} #${characterId}\n\nInformazioni aggiuntive:\n • E' stato raccolto alle: ${time}"
+
```
Comment (InZidiuZ) [inline languages/it-IT.lua:2183]: RANDOM NEWLINE?

**Review by InZidiuZ — CHANGES_REQUESTED** [review]

File: languages/it-IT.lua [hunk 4]
```diff
@@ -2462,7 +2462,7 @@ OP.Global.Locales.Languages["it-IT"] = {
 		citizen_card = "Documento Cittadino",
 		citizen_card_description = "Agisce come identificazione, patente di arma da fuoco e patente di guida",
 		phone = "Telefono",
-		phone_description = "never:tm:",
+		phone_description = "Usalo per chiamare qualche tuo amico",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:2465]: Translations should not go against their original meaning. A locale should have the exact same meaning in all languages as it should be possible for anyone playing on any server to change their language without it causing confusion.

**Review by InZidiuZ — CHANGES_REQUESTED** [review]

File: languages/it-IT.lua (hunk 4 above)
Comment (Enryy) [reply languages/it-IT.lua]: ahhhhh ok, I have translated correctly now

**Comment by InZidiuZ** [comment]
You still have random newlines in the PR as I mentioned earlier. This won't be merged until that is fixed.

**Comment by Enryy** [comment]
It should be fine now

**Comment by InZidiuZ** [comment]
...what? You just re-introduced a change I said wasn't permitted.

**Comment by Enryy** [comment]
I have edited and copied the old file, sorry my mistake

**Pull request merged**